diff --git a/node_modules/web3-bzz/lib/index.js b/node_modules/web3-bzz/lib/index.js
index 569d610..cca38a7 100644
--- a/node_modules/web3-bzz/lib/index.js
+++ b/node_modules/web3-bzz/lib/index.js
@@ -23,7 +23,7 @@
 var _ = require('underscore');
 var swarm = require("swarm-js");
 var Bzz = function Bzz(provider) {
-    console.warn('web3-bzz package will be deprecated in version 1.3.5 and will no longer be supported.');
+    // console.warn('web3-bzz package will be deprecated in version 1.3.5 and will no longer be supported.');
     this.givenProvider = Bzz.givenProvider;
     if (provider && provider._requestManager) {
         provider = provider.currentProvider;
diff --git a/node_modules/web3-bzz/src/index.js b/node_modules/web3-bzz/src/index.js
index 56cd0fe..cf0a828 100644
--- a/node_modules/web3-bzz/src/index.js
+++ b/node_modules/web3-bzz/src/index.js
@@ -29,7 +29,7 @@ var swarm = require("swarm-js");
 var Bzz = function Bzz(provider) {
 
 
-    console.warn('web3-bzz package will be deprecated in version 1.3.5 and will no longer be supported.')
+    // console.warn('web3-bzz package will be deprecated in version 1.3.5 and will no longer be supported.')
 
 
     this.givenProvider = Bzz.givenProvider;
