diff --git a/node_modules/web3-shh/lib/index.js b/node_modules/web3-shh/lib/index.js
index b636d8f..b5c8450 100644
--- a/node_modules/web3-shh/lib/index.js
+++ b/node_modules/web3-shh/lib/index.js
@@ -26,7 +26,7 @@ var Method = require('web3-core-method');
 // var formatters = require('web3-core-helpers').formatters;
 var Net = require('web3-net');
 var Shh = function Shh() {
-    console.warn('web3-shh package will be deprecated in version 1.3.5 and will no longer be supported.');
+    // console.warn('web3-shh package will be deprecated in version 1.3.5 and will no longer be supported.');
     var _this = this;
     // sets _requestmanager
     core.packageInit(this, arguments);
diff --git a/node_modules/web3-shh/src/index.js b/node_modules/web3-shh/src/index.js
index e1e893a..eec3bca 100644
--- a/node_modules/web3-shh/src/index.js
+++ b/node_modules/web3-shh/src/index.js
@@ -32,7 +32,7 @@ var Net = require('web3-net');
 var Shh = function Shh() {
 
 
-    console.warn('web3-shh package will be deprecated in version 1.3.5 and will no longer be supported.')
+    // console.warn('web3-shh package will be deprecated in version 1.3.5 and will no longer be supported.')
 
 
     var _this = this;
